<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerAnnotationModuleIOOptionsWidget</class>
 <widget class="qSlicerWidget" name="qSlicerAnnotationModuleIOOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Options</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="NameLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="FiducialRadioButton">
     <property name="text">
      <string>Fiducial</string>
     </property>
     <property name="checked">
       <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="RulerRadioButton">
     <property name="text">
      <string>Ruler</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="ROIRadioButton">
     <property name="text">
      <string>ROI</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
